### DISCUSSION TITLE
account creation on budibase.com keep failing

### DISCUSSION THREAD

**Discussion by ymolists** [Budibase/budibase#4247] [Help] [answered]
Hello team.

I tried signing up for the app online after failing to run it locally :-( it keeps sending me to `https://%20ymo000.budibase.app/` when i login now. Pray help :-)

**Comment by mjashanks** [comment, marked as answer]
Hey @ymolists - it looks like you may have put a space in your domain name (we should do a check for this, sorry!)...

`%20ymo000` = `<space>ymo000`

Are you able to log into https://account.budibase.app and delete your account? Then signup again.

**Reply by ymolists** [reply]
@mjashanks thank you so much for looking at this ! I am still getting that error even after i cleared by cache !

**Reply by mjashanks** [reply]
no problem - we're you able to delete your account?

**Reply by ymolists** [reply]
i cant delete ... because i cant login :-) it keeps taking me back to that url `https://%20ymo000.budibase.app/` no matter what i do. Its definitely a bug i am sorry i triggered this. I have a curse for these types of things !

**Reply by mjashanks** [reply]
yea - definitely an issue on our end. I don't have access to fix it internally. Do you have another email address that you could use to signup with?

If you are using gmail / google workspace, you could use the `your_email+budibase@gmail.com`  and it will still go to the same email account.

**Reply by ymolists** [reply]
Thanks for the help i am able to login using that email !

Best Regards

**Comment by ymolists** [comment]
Can someone please take a look at my account ? it still insists on redirecting me to `https://%20ymo000.budibase.app/` i would not mind if someone can just delete the actual account. I am assuming whatevet was causing the error is now fixed ?

**Reply by shogunpurple** [reply]
@ymolists Just checked and it doesn't seem like there is a space before or after your tenantId, so I'm not sure what the issue might be here - in fact the platform should not let you sign up in that case. Moreover, it seems like you've created 3 tenants.

I can see:
- `ymo000`
- `ymo000budi`
- `ymo001`

Are you now actively using any of the other ones? I can just delete the `ymo000` one if it's old. Thanks!

**Reply by ymolists** [reply]
@shogunpurple I created one with `<email>+budibase@gmail.com` after it was siggested above. But i cant get back in it as well :-( 

please remove all of them :-) i am not using any.

**Reply by shogunpurple** [reply]
Ok @ymolists I'll remove them - it's odd you can't login though, possibly try clearing all the cookies in your browser and logging in again
